src-backbone/app/js/views/auth/resetPasswordCompleteView.js

```diff
@@ -0,0 +1,69 @@
+const App = require('utils/sanaAppInstance');
+const Helpers = require('utils/helpers');
+
+module.exports = Marionette.ItemView.extend({
+
+    initialize: function(reset_token) {
```

Comment:
nit: I think we usually put `initialize` after the other definitions (in this case, `template`, `ui`, and `events`)?